src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -125,6 +128,10 @@ export default abstract class ComparisonStore
     @observable includeSomaticMutations = true;
     @observable includeUnknownStatusMutations = true;
 
+    @observable geneQuery = '';
+    @observable selectedGenes: string[] = [];
+    @observable isHighestFrequencedGenesCalculated = false;
```

Comment:
we probably don't need this flag, we can compute the initial `geneQuery` by using `genesWithMaxFrequency`